web/src/App.tsx

```diff
@@ -104,12 +104,7 @@ export default function App(): ReactElement {
               <Suspense>
                 <TimeControllerWrapper />
               </Suspense>
-              <Suspense>
-                <FeatureFlagsManager />
-              </Suspense>
-              <Suspense>
-                <LegendContainer />
-              </Suspense>
```

Comment:
No more LegendContainer? If so the dynamic import should probably be removed as well.